Automattic/jetpack
# Forms: Update inbox view responses styles

## Proposed changes:

* Updates Forms inbox view responses styles

### Other information:


## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

Enable the jetpack forms package and the new dashboard before testing with:

```
add_filter( 'jetpack_contact_form_use_package', '__return_true' );
add_filter( 'jetpack_forms_dashboard_enable', '__return_true' );
```

- Go to `/wp-admin/admin.php?page=jetpack-forms`
- Check if the Inbox responses styles match the design proposed in the Figma file
